src/content/docs/en/core-concepts/astro-components.mdx

```diff
@@ -17,7 +13,7 @@ The most important thing to know about Astro components is that they **render to
 
 ## Component Structure
 
-An Astro component is made up of two main parts: the **Component Script** and the **Component Template**. Each part performs a different job, but together they aim to provide a framework that is both easy to use and expressive enough to handle whatever you might want to build.
+An Astro component is made up of two main parts: the **Component Script** and the **Component Template**. Each part performs a different job, but together they provide a framework that is both easy to use and expressive enough to handle whatever you might want to build.
```

Comment:
Now is a good time to standardize to **frontmatter (script)**, which is used in the tutorial and more prevalent in our community.